### PR TITLE
MES-2525: Configure Env Variables

### PR DESCRIPTION
**Pull request by szabikr**
Set up serverless.yml to use certain configuration locally
Implementing branching logic in the handler

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -3,18 +3,34 @@ service: mes-documents-service
 provider:
   name: aws
   runtime: nodejs8.10
+  environment:
+    NOTIFY_HOSTNAME: http://www.example.com
+    USE_NOTIFY: false
+    IS_LOCAL: true
```
Comment (TomBillingtonUK) [inline serverless.yml:9]: could you add the other var's as empty strings for now :)

NOTIFY_EMAIL_TEMPLATE_ID
NOTIFY_API_KEY
NOTIFY_API_BASE_URL
NOTIFY_POST_TEMPLATE_ID
NOTIFY_EMAIL_WELSH_TEMPLATE_ID

File: serverless.yml [hunk 2]
```diff
@@ -3,18 +3,34 @@ service: mes-documents-service
...
+    reservedConcurrency: 1
 
 custom:
   webpack:
     webpackConfig: 'webpack-sls-offline.config.js'
+  serverless-offline-http-mock:
+    - hostname: ${self:provider.environment.NOTIFY_HOSTNAME}
+      directory: 'mocks'
+      mocks:
+        - index.js
```
Comment (TomBillingtonUK) [inline serverless.yml:32]: You haven't added this file

File: serverless.yml [hunk 3]
```diff
@@ -3,18 +3,34 @@ service: mes-documents-service
...
 
 plugins:
-  - serverless-webpack
+  - serverless-offline-http-mock
```
Comment (TomBillingtonUK) [inline serverless.yml:35]: I don't think you've installed this package (unless it was already there)

**Review by TomBillingtonUK — APPROVED** [review]
Looks good - a couple of comments

File: serverless.yml (hunk 1 above)
Comment (szabikr) [reply serverless.yml:9]: I think we could add them when we need them.

Comment (TomBillingtonUK) [reply serverless.yml:9]: both of our next sub tasks will require at least the api key- so one of us will be waiting on the other

Comment (szabikr) [reply serverless.yml:9]: okay, will add that. But really after this PR all different PRs can bring their additional env variables

File: serverless.yml (hunk 3 above)
Comment (szabikr) [reply serverless.yml:35]: installed it

File: serverless.yml (hunk 2 above)
Comment (szabikr) [reply serverless.yml:32]: added now

**Review by roberthall12 — APPROVED** [review]
looks good

**Pull request merged**